DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_SearchResultsCount.cshtml

```diff
@@ -5,5 +5,5 @@
   @{
     var count = Model.MatchingSearchResults;
   }
-  <span role="alert" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>
+  <span aria-live="polite" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>

```

Comment:
I've replaced the role="alert" with aria-live="polite" as it seems more appropriate given the description in the docs:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions